camel/agents/chat_agent.py

```diff
@@ -252,6 +239,7 @@ def update_messages(self, role: str,
         r"""Updates the stored messages list with a new message.
 
         Args:
+            role (str): Role of the agent.
```

Comment:
Thanks for spotting this lack of annotation. However this is not "Role of the agent". It is "Role of the message at backend". Procedurally you'd better make a separate bug ticket for this.

Reply:
I think fixing it here is fine. Making a separate bug ticket is too much.